ambry-commons/src/main/java/com.github.ambry.commons/BlobId.java

```diff
@@ -85,7 +85,8 @@
  * </pre>
  *
  * <br>
- * Version 4, which is the same as Version 3 now.
+ * Version 4 has the same format with Version 3. The difference is Version 3 ignores isEncrypted bit in flag while
```

Comment:
Saying that this is version 4 is enough I think - no need for the comparison with 3